src/main/java/racingcar/domain/car/Name.java

```diff
@@ -0,0 +1,41 @@
+package racingcar.domain.car;
+
+import java.util.Objects;
+import utils.StringUtils;
+
+public class Name {
+
+    public static final int MAXLENGTH = 5;
+    private String name;
+
+    public Name(String name) {
+        if (StringUtils.isNullOrBlank(name)) {
+            throw new IllegalArgumentException("이름값이 있어야 합니다.");
+        }
+        if (name.length() > MAXLENGTH) {
+            throw new IllegalArgumentException("이름이 5자 이하만 가능합니다.");
```

Comment:
```suggestion
            throw new IllegalArgumentException("이름이 " + MAXLENGTH + "자 이하만 가능합니다.");
```

상수가 변경 됐을 때 예외처리 메시지도 신경쓰는 문제를 제거해볼 수 있겠네요!